api/v1/views/cities.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+from api.v1.views import app_views
+from flask import Flask, jsonify, abort, make_response, request
+from models import storage
+from models.state import State
+from models.city import City
+
+
+@app_views.route('/states/<state_id>/cities',
+                 methods=['GET'], strict_slashes=False)
+def get_cities_states_id(state_id):
```

Comment:
saw in other scripts that this definition could use a better name. Something like "get_state_id_cities" It makes more sense regarding the route.

Reply:
Change fixed